tests/fuzz.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
```

Comment:
Remove shebang if we're running it via `python tests/fuzz.py`?
```suggestion
```

Reply:
Either that or making the file `+x`, but that'd differ from the other scripts in `tests/`, so I agree.